picard/ui/coverartbox.py

```diff
@@ -75,33 +75,40 @@ def dropEvent(self, event):
         # is useful for Google Images, where the url links to the page that contains the image
         # so we use it if the downloaded url is not an image.
         mime_data = event.mimeData()
-        dropped_data = mime_data.data('application/octet-stream')
+        dropped_data = bytes(mime_data.data('application/octet-stream'))
 
         if not dropped_data:
-            dropped_data = mime_data.data('application/x-qt-image')
+            dropped_data = bytes(mime_data.data('application/x-qt-image'))
 
-        try:
-            mime = imageinfo.identify(dropped_data)[2]
-            if mime in ('image/jpeg', 'image/png'):
-                accepted = True
-                self.image_dropped.emit(QtCore.QUrl(''), dropped_data)
-        except imageinfo.IdentificationError:
-            pass
+        if dropped_data:
+            try:
+                mime = imageinfo.identify(dropped_data)[2]
+                if mime in ('image/jpeg', 'image/png'):
+                    accepted = True
+                    log.debug("Dropped %s mime data (%d bytes)", mime,
+                              len(dropped_data or ''))
+                    self.image_dropped.emit(QtCore.QUrl(''), dropped_data)
+            except imageinfo.IdentificationError:
+                pass
```

Comment:
I think we should log this as a warning, at minimum debug. We have enough totally ignored exceptions already :D

Reply:
Addressed in 71737ed